### PR TITLE
Update gradle version to 2.1.2

### PR DESCRIPTION
**Pull request by geralt-encore**
Updated gradle version, because 1.5 is quite outdated.
Sdk manager plugin doesn't work with gradle 2.0+ ([source](https://github.com/JakeWharton/sdk-manager-plugin/issues/99)). Maybe there is a better solution?

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Let's remove SDK manager plugin, it doesn't really work and deprecated by Jake.

File: build.gradle [hunk 1]
```diff
@@ -3,14 +3,12 @@
 buildscript {
     repositories {
         jcenter()
+        maven { url 'https://jitpack.io' }
```
Comment (nikitin-da) [inline build.gradle]: It seems it is not required ;)

**Comment by nikitin-da** [comment]
👍

**Comment by artem-zinnatullin** [comment]
👍

**Comment by nikitin-da** [comment]
Thanks, @geralt-encore!

**Comment by oldergod** [comment]
@artem-zinnatullin How will you deal with CI now that you removed the SDK manager ?

**Comment by artem-zinnatullin** [comment]
Just declare all dependencies in .travis.yml/circle.yml

**Pull request merged**